### PR TITLE
Support transpose by merge Reshape etc into direct xint8 operators.

### PR DESCRIPTION
**Pull request by zhanghuanrong**
* add quantized operator for those ops supprot xint8 intput[0]
 * merge reshape/maxpool to utilize above base
 * support transpose through above

### CODE REVIEW

File: onnxruntime/test/python/quantization/test_op_reshape.py [hunk 1]
```diff
@@ -76,12 +75,13 @@ def test_quantize_reshape(self):
                                   [3, 7],
                                   [7, 3],
                                   [1, 9])
-        quantize_static(model_fp32_path,
-                        model_uint8_path,
-                        data_reader
-                        )
+        quantize_static(model_fp32_path, model_uint8_path, data_reader)
         data_reader.rewind()
         qdq_nodes = {'QLinearMatMul': 1, 'QuantizeLinear': 1, 'DequantizeLinear': 1, 'Reshape': 1}
```
Comment (yufenglee) [inline onnxruntime/test/python/quantization/test_op_reshape.py:80]: >qdq_nodes [](start = 8, length = 9)

nit: name is not correct.

Comment (yufenglee) [reply onnxruntime/test/python/quantization/test_op_reshape.py:80]: And could you please add test for QDQ format

---
In reply to: [609102392](https://github.com/microsoft/onnxruntime/pull/7265#discussion_r609102392) [](ancestors = 609102392)

File: onnxruntime/test/python/quantization/test_op_transpose.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    def test_quantize_transpose(self):
+        np.random.seed(1)
+        model_fp32_path = 'transpose_fp32.onnx'
+        model_uint8_path = 'transpose_uint8.onnx'
+        data_reader = self.input_feeds(1, {'input': [3, 7]})
+        self.construct_model_matmul_transpose(model_fp32_path, [3, 7], [7, 5], [5, 3])
+        quantize_static(model_fp32_path, model_uint8_path, data_reader)
+
+        data_reader.rewind()
+        qdq_nodes = {'QLinearMatMul': 1, 'QuantizeLinear': 1, 'DequantizeLinear': 1, 'Transpose': 1}
```
Comment (yufenglee) [inline onnxruntime/test/python/quantization/test_op_transpose.py:74]: >qdq_nodes [](start = 8, length = 9)

nit: similar issue. And please add a test for QDQ format.

File: onnxruntime/test/python/quantization/test_op_reshape.py (hunk 1 above)
Comment (zhanghuanrong) [reply onnxruntime/test/python/quantization/test_op_reshape.py:80]: cool

**Review by yufenglee — APPROVED** [review]
:shipit:

**Pull request merged**